llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -1328,8 +1328,8 @@ void PassBuilder::addVectorPasses(OptimizationLevel Level,
     ExtraPasses.addPass(
         SimplifyCFGPass(SimplifyCFGOptions().convertSwitchRangeToICmp(true)));
     ExtraPasses.addPass(InstCombinePass());
-    FPM.addPass(std::move(ExtraPasses));
   }
+  FPM.addPass(std::move(ExtraPasses));
```

Comment:
Why this move?

Reply:
This was a consequence of using a single marker, this required the pass manager to run unconditionally to clear the marker set by LV.